views/about.lavender

```diff
@@ -24,7 +24,7 @@ body
       .navbar-collapse.collapse
         ul.nav.navbar-nav
           li
-            a(href="/lavender.html", title="Home") Home
+            a(href="/index.html", title="Home") Home
```

Comment:
I'm not sure that you need the '/' at the beginning of these paths. With them, I get 404s. Without, the links work.